_includes/javascripts.html

```diff
@@ -0,0 +1,6 @@
+<script src="{{ "/js/bower_components/lodash/dist/lodash.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/graphlib/dist/graphlib.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/dagre/dist/dagre.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/dagre-d3/dist/dagre-d3.core.js" | prepend: site.baseurl }}"></script>
+<script src="{{ "/js/bower_components/d3/d3.js" | prepend: site.baseurl }}"></script>
```

Comment:
TODO: Add a dependency to Grunt to build these into one central `vendor.js` file